specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json

```diff
@@ -0,0 +1,369 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SipRouting Service",
+    "description": "SipRouting Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2022-10-01-preview"
+  },
+  "paths": {
+    "/sip": {
+      "get": {
+        "tags": [
+          "SipConfiguration"
+        ],
+        "summary": "Gets SIP configuration for resource.",
+        "operationId": "SipRouting_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/SipConfiguration"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get SIP configuration for resource": {
+            "$ref": "./examples/GetSipConfiguration.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "SipConfiguration"
+        ],
+        "summary": "Patches SIP configuration for resource.",
+        "operationId": "SipRouting_Patch",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Sip configuration patch object.",
+            "schema": {
+              "$ref": "#/definitions/SipConfigurationPatch"
```

Comment:
So this, and any containing types, would change to `SipConfigurationProperties`.